### ISSUE TITLE
Rest API

### ISSUE THREAD

**Issue by Zyles** [pagekit/pagekit#206]
Interesting project.

What about a REST API? Maybe you can utilize FOSRestBundle?

My use case: What I am trying to find is a lightweight CMS that I can make distributed. Which means one centralized control panel to connect to multiple CMS on different servers. With an API this would be possible.

I could technically do it with Wordpress, but it is not lightweight and I want to use SQLite by default for faster deployment and avoid the database setup hassle until I actually need it. And I kinda want a more modern platform to work with.

**Comment by saschadube** [comment]
We released the [Pagekit Beta](http://www.pagekit.com/blog/2015/09/10/pagekit-beta-released) today. I close this issue because the code base completely changed. Please open a new issue if it still exists.

**Comment by AdrianoCahete** [comment]
Is Pagekit builded with a REST API in this release?

**Comment by laustdeleuran** [comment]
+1 - What is the correct way to do this in [PageKit v1](https://pagekit.com/blog/2016/04/13/pagekit-1-0-is-here)?

**Comment by MalteScharenberg** [comment]
Basically most of Pagekits backend functionality is done via REST calls. What is currently missing is a stateless authentication (https://github.com/pagekit/pagekit/issues/614).

**Comment by laustdeleuran** [comment]
@MalteScharenberg Okay, so the functionality is there, that's great news. I personally don't need authentication (I think), since I just need the API for consumption. But [reading through](https://pagekit.com/docs/developer-guides/themes) [the docs](https://pagekit.com/docs/developer-basics/views-templating), I don't see any mention of the API. Am I missing something?

**Comment by MalteScharenberg** [comment]
@laustdeleuran It is not documented yet, because we only used it for Pagekits admin panel scripts and that is also the reason why most of the calls need authentication because they were designed specifically for the admin panel. What do you want to achieve with API calls?

**Comment by AdrianoCahete** [comment]
> What do you want to achieve with API calls? 

I think what most people want: Integration with mobile. Iconic, for example.

But for that, we neeed an API who can access public data.

**Comment by laustdeleuran** [comment]
@MalteScharenberg @AdrianoCahete Yeah, integration with mobile, and basic SPAs (AngularJS, React/Redux). So much easier to digest API-level data after the initial page load. 

Really, I think theming should be decoupled from most CMS as a plugin or similar. Would make the majority of CMS much more flexible. But that is kinda off-topic.

**Comment by GlauberF** [comment]
Hello,
you have the api module , which is made the REST calls to available I pay?
